### ISSUE TITLE
Az sig create version fails with error  'NoneType' object is not subscriptable - Azcli 2.17.1

### ISSUE THREAD

**Issue by pumpkinclarinet** [Azure/azure-cli#16584]
## Describe the bug

**Command Name**
`az sig image-version create`

**Errors:**
```
'NoneType' object is not subscriptable
Traceback (most recent call last):
python3.6/site-packages/knack/cli.py, ln 215, in invoke
    cmd_result = self.invocation.execute(args)
cli/core/commands/__init__.py, ln 659, in execute
    raise ex
cli/core/commands/__init__.py, ln 722, in _run_jobs_serially
    results.append(self._run_job(expanded_arg, cmd_copy))
...
azure/cli/core/__init__.py, ln 784, in default_command_handler
    return op(**command_args)
cli/command_modules/vm/custom.py, ln 3276, in create_image_version
    external_bearer_token = external_tokens[0][0] + ' ' + external_tokens[0][1]
TypeError: 'NoneType' object is not subscriptable
```

## To Reproduce:
Steps to reproduce the behavior. Note that argument values have been redacted, as they may contain sensitive information.

- _Put any pre-requisite steps here..._
- `az sig image-version create -g {} --gallery-name {} --gallery-image-definition {} --end-of-life-date {} --gallery-image-version {} --target-regions {} {} --replica-count {} --managed-image {} --verbose`

## Environment Summary
```
Linux-4.15.0-1103-azure-x86_64-with-debian-10.2 (Cloud Shell)
Python 3.6.10
Installer: DEB

azure-cli 2.17.1 *
azure-cli-ml 1.19.0

Extensions:
ai-examples 0.2.5
azure-cli-ml 1.19.0

```

**Comment by pumpkinclarinet** [comment]
This can be seen on Azure CloudShell since the version is that one.
But testing for az 2.18 this does not occur.

**Comment by yungezz** [comment]
https://github.com/Azure/azure-cli/issues/14287

**Comment by yungezz** [comment]
hi @qwordy is this the bug fixed in 2.18.0?

**Comment by qwordy** [comment]
Hi @pumpkinclarinet , it is fixed in 2.18.0. I tried Azure CLI in CloudShell just now. It is 2.18.0 now.

**Comment by qwordy** [comment]
> 
> 
> hi @qwordy is this the bug fixed in 2.18.0?

Yes, it is.
